src/System.Diagnostics.Process/src/System/Diagnostics/ProcessThread.OSX.cs

```diff
@@ -13,8 +13,11 @@ public partial class ProcessThread
         /// </summary>
         private ThreadPriorityLevel PriorityLevelCore
         {
-            get { throw new PlatformNotSupportedException(); }
-            set { throw new PlatformNotSupportedException(); }
+            // We couldn't find a POSIX API on OSX
```

Comment:
Yes, I couldn't find a posix api to do this on osx. Considered the posix pthread_getschedparam, and pthread_setschedparam, but those seems to specify the scheduling policy with the priority.